CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Note: This is in reverse chronological order, so newer entries are added to the
 Swift 5.8
 -----------
 
+* [#5966]
```

Comment:
Since this isn't merged into 5.8 branch yet, would it make sense to add a section at the top for the next version and move this item there?

Reply:
ok

Reply:
I will move it there with the next change. I was hoping to have some opinions about which alternative to `-Xhost` to use, but I cannot find a better alternative in my mind.